examples/regression/plot_timeseries_enbpi_train.py

```diff
@@ -0,0 +1,81 @@
+"""
```

Comment:
Move this script into a function within the main example which can be called with a boolean ?